keyboards/planck/keymaps/raffle/config.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace these two lines with just `#pragma once`, and remove the `#endif` at the end?